### DISCUSSION TITLE
highlight doesn't work

### DISCUSSION THREAD

**Discussion by Flayzera** [webdevnerdstuff/vue-code-block#48] [Q&A]
When the key of an object is optional, the highlight does not work

Example: 

![Captura de Tela 2024-04-23 às 13 34 28](https://github.com/webdevnerdstuff/vue-code-block/assets/91757236/a856dd11-a270-4b19-94ba-51a117c7203f)

**Comment by Flayzera** [comment]
I ended up realizing now that the value of the keys when there is something later the same thing happens.

Example:

![Captura de Tela 2024-04-23 às 13 47 44](https://github.com/webdevnerdstuff/vue-code-block/assets/91757236/26833920-a789-4c2e-ab24-a95e7aaf9dd4)

I think 'Interface' is not highlighted due to the problem mentioned above

**Comment by webdevnerdstuff** [comment]
I would need more information to understand what you are doing. An example of the code you are using to get this result is needed as I can only assume what you are doing. If you would put together a working example (StackBlitz, CodePen, etc) that would be the most helpful to determine if it's something with this plugin, or if it's an issue with Prism/Hightlight.js (which is not something I can fix).

Also please use code blocks over screenshots, it's easier to copy paste if testing is needed.
